plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -24,6 +24,13 @@
 #include <string.h>
 
 #include "fu-plugin-vfuncs.h"
+#include "libflashrom.h"
+
+#define SELFCHECK_TRUE 1
+
+struct flashrom_flashctx *flashctx = NULL;
+struct flashrom_layout *layout = NULL;
+struct flashrom_programmer *flashprog = NULL;
```

Comment:
I would think these should be static values in `FuPluginData` not globals right?